content/en/developers/integrations/python.md

```diff
@@ -3,21 +3,11 @@ kind: documentation
 title: Install the Datadog Agent Integration Developer Tool
 description: Install the Datadog Agent Integration Developer Tool.
 ---
-This document covers how to setup a Python environment to work on Agent-based Integrations, including installing the interpreter and ensuring all of the required dependencies are present. Alternatively, you can use the standalone binaries and installers provided with each release.
-
-## GitHub releases
-
-Each [release](https://github.com/DataDog/integrations-core/releases?q=ddev-&expanded=true) provides the following:
-
-- Standalone binaries for Linux, Windows, and macOS
-- Windows AMD64 (64-bit) MSI installer
-- Windows x86 (32-bit) MSI installer
-- Windows universal (AMD64+x86) EXE installer
-- macOS DMG installer
+This document covers how to set up a Python environment to work on Agent-based Integrations, including installing the interpreter and ensuring all of the required dependencies are present. Alternatively, you can use the standalone binaries and installers provided with each release.
 
 ## Install Python
 
-Many operating systems come with a pre-installed version of Python. However, the version of Python installed by default may be older than the version used in the Agent, and may lack some required tools and dependencies. To ensure that you've everything you need to get an integration running, install a dedicated Python interpreter.
+Many operating systems come with a pre-installed version of Python. However, the version of Python installed by default may be older than the version used in the Agent, and may lack some required tools and dependencies. To ensure that you have everything you need to get an integration running, install a dedicated Python interpreter.
```

Comment:
Rephrase to `However, Python version installed by default may be older than the version used in the Agent, and may lack some required tools and dependencies.` to limit sentence with 25 words or fewer.